components/brave_rewards/browser/database/publisher_info_database.cc

```diff
@@ -706,6 +675,23 @@ std::string PublisherInfoDatabase::GetSchema() {
 
 // Migration -------------------------------------------------------------------
 
+bool PublisherInfoDatabase::MigrateV0toV1() {
```

Comment:
question: `V0`? What is / Why would version zero exist?

Reply:
version 0 is when database was not created yet, meaning that we need to start from scratch an create version 1

Reply:
Ok - was just puzzled that we'd need to migrate if there was no existing DB to migrate from.